src/purchases.ts

```diff
@@ -218,15 +245,29 @@ export default class Purchases {
       throw new Error("appUserID needs to be a string");
     }
 
+    let purchasesCompletedByToUse: PURCHASES_ARE_COMPLETED_BY_TYPE;
+    let storeKitVersionToUse = storeKitVersion;
+
+    if (Purchases.isPurchasesAreCompletedByMyApp(purchasesAreCompletedBy)) {
+      purchasesCompletedByToUse = PURCHASES_ARE_COMPLETED_BY_TYPE.MY_APP;
+      storeKitVersionToUse = (
+        purchasesAreCompletedBy as PurchasesAreCompletedByMyApp
+      ).storeKitVersion;
```

Comment:
I had a log message here if `storeKitVersionToUse` !== `storeKitVersion`, but our linter doesn't allow console.log or console.warn 😅

Reply:
Hmm right... it's usually a bad practice to leave logs (not configurable) in a TS library... However, I think in this case it might be ok. Can we just disable the linter for this log?